### PR TITLE
PC-807

### PR DESCRIPTION
**Pull request by eyalvo-matrix**

### CODE REVIEW

File: modules/Imaging/Module.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+    /**
+     * @return array
+     */
+    public function getServiceConfig()
+    {
+        return array(
+            'factories' => array(
+                ApiBuilder::class =>  function(ContainerInterface $container) {
```
Comment (matrix-amiel) [inline modules/Imaging/Module.php]: remove ApiBuilder

File: modules/Imaging/Module.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see
+ * http://www.gnu.org/licenses/licenses.html#GPL
+ *    @author  Oshri Rozmarin <oshri.rozmarin@gmail.com>
+ * +------------------------------------------------------------------------------+
+ *
+ */
+namespace Imaging;
+
+use ClinikalAPI\Service\Imaging;
```
Comment (matrix-amiel) [inline modules/Imaging/Module.php]: delete unnecessary line

File: modules/Imaging/config/module.config.php [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see
+ * http://www.gnu.org/licenses/licenses.html#GPL
+ *    @author  Eyal Wolanowski <eyal.wolanowski@gmail.com>
+ * +------------------------------------------------------------------------------+
+ *
+ */
+
+use ClinikalAPI\Controller\ManagerApi;
```
Comment (matrix-amiel) [inline modules/Imaging/config/module.config.php]: remove ManagerAPI

File: modules/Imaging/config/module.config.php [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+use Interop\Container\ContainerInterface;
+
+return array(
+
+    /* declare all controllers */
+
+    'controllers' => array(
+        'factories' => [
+            ManagerApi::class => function (ContainerInterface $container) {
```
Comment (matrix-amiel) [inline modules/Imaging/config/module.config.php]: remove ManagerAPI

**Review by matrix-amiel — COMMENTED** [review]
Review finished

File: modules/GenericBase/sql/install.sql [hunk 5]
```diff
@@ -0,0 +1,100 @@
+--
+-- Table structure for table `moh_import_data`
+--
+
+DROP TABLE IF EXISTS `moh_import_data`;
```
Comment (matrix-amiel) [inline modules/GenericBase/sql/install.sql]: move to importData module in the clinikal-backend repo

File: modules/GenericBase/sql/install.sql [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+(18, 'ICD9', 'codes', 'icd9', 'EDM', '2017-03-02 09:11:01'),
+(19, 'MOH_KUPAT_CHOLIM_BRANCHES', 'MOH_KUPAT_CHOLIM_BRANCHES', 'mkcb', 'CSV', '2017-03-29 00:00:00');
+
+-- --------------------------------------------------------
+
+--
+-- Table structure for table `moh_import_data_log`
+--
+
+DROP TABLE IF EXISTS `moh_import_data_log`;
```
Comment (matrix-amiel) [inline modules/GenericBase/sql/install.sql]: move to importData module in the clinikal-backend repo

File: modules/GenericBase/sql/install.sql [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+INSERT INTO `categories_seq` (`id`) VALUES('9');
+
+
+-- FOR EVERY LIST FROM LIST OPTIONS WE USED IN THE FHIR API/ CLINIKAL API WE NEED ADD -
+-- FOR EXAMPLE - INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `option_value`, `mapping`, `notes`, `codes`, `toggle_setting_1`, `toggle_setting_2`, `activity`, `subtype`, `edit_options`, `timestamp`) VALUES ('lists','mh_cities','moh cities',311,1,0,'','','',0,0,1,'',1,'2017-03-02 07:07:44');
+
+
+-- FOR EVERY NEW COLUNMS IN PATIENT_DATA WE USED IN THE FHIR API/ CLINIKAL API WE NEED ADD -
+
+--ALTER TABLE `patient_data`
```
Comment (matrix-amiel) [inline modules/GenericBase/sql/install.sql]: You were tired....

File: modules/GenericBase/sql/install.sql [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+DELETE FROM `categories` WHERE `id` = '7';
+DELETE FROM `categories` WHERE `id` = '29';
+DELETE FROM `categories` WHERE `id` = '26';
+DELETE FROM `categories` WHERE `id` = '23';
+
+DELETE FROM `categories_seq` WHERE `id` = '29';
+INSERT INTO `categories_seq` (`id`) VALUES('9');
+
+
+-- FOR EVERY LIST FROM LIST OPTIONS WE USED IN THE FHIR API/ CLINIKAL API WE NEED ADD -
```
Comment (matrix-amiel) [inline modules/GenericBase/sql/install.sql]: You were tired....

Comment (matrix-amiel) [reply modules/GenericBase/sql/install.sql]: and nee to be in the clinikalApi as I wrote

File: modules/Imaging/Module.php (hunk 1 above)
Comment (eyalvo-matrix) [reply modules/Imaging/Module.php]: ok

File: modules/Imaging/Module.php (hunk 2 above)
Comment (eyalvo-matrix) [reply modules/Imaging/Module.php]: ok

File: modules/Imaging/config/module.config.php (hunk 3 above)
Comment (eyalvo-matrix) [reply modules/Imaging/config/module.config.php]: ok

File: modules/Imaging/config/module.config.php (hunk 4 above)
Comment (eyalvo-matrix) [reply modules/Imaging/config/module.config.php]: ok

File: modules/GenericBase/sql/install.sql (hunk 5 above)
Comment (eyalvo-matrix) [reply modules/GenericBase/sql/install.sql]: ok

File: modules/GenericBase/sql/install.sql (hunk 7 above)
Comment (eyalvo-matrix) [reply modules/GenericBase/sql/install.sql]: ok

File: modules/GenericBase/sql/install.sql (hunk 6 above)
Comment (eyalvo-matrix) [reply modules/GenericBase/sql/install.sql]: ok

File: modules/GenericBase/sql/install.sql (hunk 8 above)
Comment (eyalvo-matrix) [reply modules/GenericBase/sql/install.sql]: Just this line ?

**Pull request merged**